exonum/src/node/consensus.rs

```diff
@@ -613,16 +680,31 @@ impl NodeHandler {
     /// Handles external boxed transaction. Additionally transaction will be broadcast to the
     /// Node's peers.
     #[cfg_attr(feature = "cargo-clippy", allow(clippy::needless_pass_by_value))]
-    pub fn handle_incoming_tx(&mut self, msg: Verified<AnyTx>) {
+    pub(crate) fn handle_incoming_tx(&mut self, msg: Verified<AnyTx>) {
```

Comment:
Nit / future work: Could make sense to have a sandbox test for an invalid transaction.